Description: Fix unicode strings in pbr/tests/test_setup.py
 Avoids failure in Python 3.2.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-04-11

--- a/pbr/tests/test_setup.py
+++ b/pbr/tests/test_setup.py
@@ -20,6 +20,7 @@
 import sys
 import tempfile
 import testscenarios
+import six
 
 try:
     import cStringIO as io
